### ISSUE TITLE
Enable Pro features

### ISSUE THREAD

**Issue by lizconlan** [mysociety/asktheeu-theme#77]
Depends on #66 

Part of https://github.com/mysociety/transparency-adessium/issues/3

## Setup / config changes (to do before launch)
- [x] [Enable POP polling](https://github.com/mysociety/sysadmin/issues/1158)

## Must haves

- [x] Upgrade to 0.35 https://github.com/mysociety/asktheeu-theme/issues/97
- [x] Upgrade to 0.36 https://github.com/mysociety/asktheeu-theme/issues/100
- [x] Check that the site will run with Pro switched on
- [x] Check that the site will run with Pro Pricing switched on
- [x] [Allow pricing to be enabled without manually flipping the feature flag](https://github.com/mysociety/alaveteli/pull/5291)
- [x] Check that you can subscribe to a plan and get the correct permissions & feature flags
- [x] Ensure all the correct role records exist
- [x] Create `pro`, `pro_admin` roles
- [x] Check that you can cancel subscription

#### Theme level files
- [x] Ask for legal T&Cs copy 
- [x] Add legal T&Cs page https://github.com/mysociety/asktheeu-theme/issues/96
- [x] [Update the `help/requesting` page to promote the Pro embargoes feature](https://github.com/mysociety/asktheeu-theme/issues/95)
- [x] Add `/help/pro`
- [x] Add `/help/pro` translations

#### Transifex
- [x] [Make the plan page translatable](https://github.com/mysociety/alaveteli/pull/5295)
- [x] [Make the pricing page translatable](https://github.com/mysociety/alaveteli/pull/5294)
- [x] [Fixes for the promo page](https://github.com/mysociety/alaveteli/pull/5317)
- [x] [Make release so that we can ask for the strings to be translated](https://github.com/mysociety/alaveteli/issues/5229)

### Stripe
- [x] [Set the Stripe API version in code](https://github.com/mysociety/alaveteli/pull/5298)
- [x] [Update the Stripe gem version (newer ruby library)](https://github.com/mysociety/alaveteli/pull/5299)
- [x] [Rake task to create the webhook endpoint](https://github.com/mysociety/alaveteli/pull/5315)
- [x] Ask site admins to create and set up their Stripe account
  - [x] Create API keys and hand to mySociety
  - [x] Create pro plan
- [x] Set up the webhook endpoint

### Documentation
- [x] Pro help page - for site users
  - [x] WhatDoTheyKnow image override https://github.com/mysociety/whatdotheyknow-theme/pull/585
  - [x] Stub in core https://github.com/mysociety/alaveteli/pull/5292
  - [x] Alavetelitheme (example) override https://github.com/mysociety/alavetelitheme/pull/110
  - [x] AskTheEU override https://github.com/mysociety/asktheeu-theme/pull/94
  - [x] WhatDoTheyKnow override https://github.com/mysociety/whatdotheyknow-theme/pull/586
- [x] [Pro Pricing config docs - for site admins](https://github.com/mysociety/alaveteli/pull/5290) (gh-pages)

## Nice to have
- [x] Daily webhook summary emails
- [x] Weekly Pro Metrics email
- [x] Generic "none" theme versions of `alaveteli_pro/screenshot-dashboard.jpg` and `alaveteli_pro/screenshot-requests.jpg`

**Comment by lizconlan** [comment]
Likely to be some hidden complexity here around pricing (Stripe setup, pricing levels, currency), and possibly around the `/pro` and `/pro/pricing` pages

**Comment by lizconlan** [comment]
## Must haves

- [x] Check that the site will run with Pro switched on
- [x] Check that the site will run with Pro Pricing switched on
- [ ] Allow pricing to be enabled without manually flipping the feature flag

### Translations
#### Theme level files
- [ ] Ask for legal Ts&Cs copy 

#### Transifex
- [ ] Make the plan page translatable
- [ ] Make the pricing page translatable
- [ ] Make release so that we can ask for the strings to be translated

### Stripe
- [ ] Rake task to create the webhook
- [ ] Set the Stripe API version
- [ ] Update the Stripe gem version (newer ruby library)

### Documentation

## Nice to have
- [x] Daily webhook summary emails
- [x] Weekly Pro Metrics email
